docs/concepts/controller-manager.md

```diff
@@ -98,8 +98,10 @@ In order to activate it, provide the following configuration:
 * `concurrentSyncs`: The amount of goroutines scheduled for reconciling events.
 * `ttlNonShootEvents`: When an event reaches this time-to-live it gets deleted unless it is a Shoot-related event (defaults to `1h`, equivalent to the `event-ttl` default).
 
-> :warning: In addition, you should also configure the `--event-ttl` for the kube-apiserver to define an upper-limit of how long Shoot-related events should be stored.
+{{% alert color="warning"  title="Warning" %}}
```

Comment:
What is this special syntax? Can we please retain the markdown syntax?

Reply:
We have moved away from using the markdown syntax in the documentation due to the increased visibility and functionality the shortcodes offer in the website. 

Here is how the rendered markdown looks:
![Screenshot 2022-11-03 111057](https://user-images.githubusercontent.com/25197046/199684586-09e1c2be-1ce9-426c-a5c8-00d3516da12c.png)
And here is how the rendered shortcode:
![Screenshot 2022-11-03 111229](https://user-images.githubusercontent.com/25197046/199684593-55542e67-5517-468d-9b4e-119179b0fcc6.png)

You can find more information on the subject of Shortcodes [in the documentation](https://gardener.cloud/docs/contribute/20_documentation/30_shortcodes/).

Reply:
Given that all these docs are written and maintained by developers and we don't have dedicated folks just for the documentation, I doubt that anything else other than Markdown-only is a sustainable way. You don't want to run behind all these changes and adapt it again and again.
Personally, I don't see much difference in the display, so the impact of remaining with Markdown seems rather minimal.